torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -8,21 +8,20 @@ namespace torch {
 namespace jit {
```

Comment:
good cleanup below, but please isolate in separate PRs.

Reply:
This small rewrite was required since we removed IRInputOutput.